docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
```

Comment:
This file is used in local test of vectorcxx dev image, will not be used in CI process.